### DISCUSSION TITLE
Como obter o conteúdo pesquisado ao utilizar o PoPageDynamicTableComponent

### DISCUSSION THREAD

**Discussion by gtrabach** [po-ui/po-angular#1698] [Tire suas dúvidas] [answered]
Ao utilizar o componente PoPageDynamicTableComponent (Page Dynamic Table), como posso obter o conteúdo pesquisado na lupa por um usuário?

Estou com uma demanda onde preciso exportar os dados da tabela para Excel, porém para exportar os dados da tabela filtrada, eu precisaria saber qual conteúdo o usuário utilizou nos filtros do componente.

Poderiam estar me auxiliando em como eu posso obter esses dados?

Observação: Podem me sugerir outras formas também de como realizar esse processo de exportação sem a necessidade de buscar pelo filtro utilizado.

**Comment by anliben** [comment]
Boa Tarde @gtrabach, tudo bem?

Consegue montar um exemplo para eu entender melhor?

stackblitz: https://stackblitz.com/edit/po-ui

Attr.

**Reply by gtrabach** [reply]
Bom dia @anliben.

Tentei montar um exemplo mas tá exibindo o seguinte erro: Property 'downloadService' is used before its initialization.

Poderia me auxiliar? https://stackblitz.com/edit/po-ui-mb42qy

**Reply by anliben** [reply, marked as answer]
Boa Tarde @gtrabach, tudo bem?

Estamos analisando e em breve retornamos uma resposta!

Attr.

**Reply by gtrabach** [reply]
Boa tarde, @anliben. Conseguiu checar essa situação?

**Reply by anliben** [reply]
Bom Dia @gtrabach, estamos buscando uma forma de fazer, porem em nosso componente não ha uma forma nativa de ser feito!

Attr.
